recipe/migrations/osx_arm64.txt

```diff
@@ -72,3 +72,5 @@ cctbx-base
 wxpython
 hdf5-external-filter-plugins
 mamba
+wrapt
```

Comment:
This is already in the list.

Reply:
You are right. I missed that, as I was trying to install a previous version, not available for osx-arm64.